services/ui-src/src/components/tables/getEntityStatus.tsx

```diff
@@ -118,6 +119,15 @@ export const getInitiativeStatus = (
       return getInitiativeDashboardStatus(step, entity);
     });
 
+    /**
+     * Currently, on exporting a PDF of the Work Plan does not take into consideration the Close-out initiatives entity step,
+     * but the status is always returning "incomplete" because that step is used to calculate the initiative status.
+     * This removes the Close-out initiative step status from the PDF statusing calculation.
+     */
+    if (isPdf) {
+      stepStatuses.splice(-1);
+    }
```

Comment:
thought: I understand this works for now. Will this always be the case? Is there a way we can check if the close-out step should be counted and use that instead of always removing it? again, maybe not relevant atm

Reply:
as a follow-on: we could add `entity?.isInitiativeClosed` to this if condition?

Reply:
@gmrabian I added a second condition based on @ajaitasaini 's suggestion: that final step will only be relevant if the Work Plan is a copy-over, so now I'm checking whether it's a PDF and if it's been copied over.